### DISCUSSION TITLE
Awarding the best AdGuard contributors in 2022

### DISCUSSION THREAD

**Discussion by aalpatkina** [AdguardTeam/AdGuardHome#5263] [General]
Hello, @jslawler!

My name is Anna, I'm a content manager at AdGuard. Currently we’re preparing a blog post about the most active community contributors of the year, and you undoubtedly deserve a spot there. Congratulations and big thanks for your help!

Please contact me at [aalpatkina@adguard.com](mailto:aalpatkina@adguard.com) to discuss the best way to send you our special gift bag and other details.

**Comment by ghost** [comment]
Hi Anna,

I've emailed you with this discussion number as the subject title.
